cadquery/cqgi.py

```
@@ -414,33 +413,30 @@ def add_entry(self, name, value):
     def build(self):
         return self.env
 
-
 class ParameterDescriptionFinder(ast.NodeTransformer):
     """
     Visits a parse tree, looking for function calls to describe_parameter(var, description )
     """
-
     def __init__(self, cq_model):
         self.cqModel = cq_model
 
-    def visit_Call(self, node):
-        """
-        Called when we see a function call. Is it describe_parameter?
-        """
-        try:
+    def visit_Call(self,node):
+       """
+       Called when we see a function call. Is it describe_parameter?
+       """
+       try:
             if node.func.id == 'describe_parameter':
                 # looks like we have a call to our function.
                 # first parameter is the variable,
                 # second is the description
                 varname = node.args[0].id
                 desc = node.args[1].s
-                self.cqModel.add_parameter_description(varname, desc)
+                self.cqModel.add_parameter_description(varname,desc)
 
-        except:
-            print("Unable to handle function call")
+       except:
```

Do we want too catch all exceptions here?